testing/jormungandr-integration-tests/src/jormungandr/vit/private.rs

```diff
@@ -182,46 +194,46 @@ pub fn jcli_e2e_flow_private_vote() {
 
     time::wait_for_epoch(3, jormungandr.explorer());
 
-    let vote_tally = jormungandr.rest().inner().vote_plan_statuses().unwrap();
-    let vote_tally_file = NamedTempFile::new("vote_tally.yaml").unwrap();
-    vote_tally_file.write_str(&vote_tally).unwrap();
+    let vote_tally = jormungandr.rest().vote_plan_statuses().unwrap();
+    let vote_tally_file = NamedTempFile::new("vote_tally_proposal_0.yaml").unwrap();
 
-    let _decryption_share = jcli
+    let encrypted_tally = match vote_tally
+        .get(0)
+        .unwrap()
+        .proposals
+        .get(0)
+        .unwrap()
+        .tally
+        .as_ref()
+        .unwrap()
+    {
+        jormungandr_lib::interfaces::Tally::Private { state } => match state {
+            jormungandr_lib::interfaces::PrivateTallyState::Encrypted {
+                encrypted_tally,
+                total_stake: _,
+            } => serde_json::to_string(&encrypted_tally).unwrap(),
```

Comment:
This adds unnecessary quotes around the output. `jcli` expects plain `base64` here.